Restore invisible brackets around nicknames (textual6 version)

dgw opened the pull request:
When copying and pasting chat logs from Textual's window, the brackets help keep the text readable.

They were silently removed during the major overhaul in 796d320. Because they weren't mentioned at all in the commit message, my thought is that it might have been accidental—especially since the CSS was left in.

This is the Textual 6 version of the patch. The version created for the Textual 5 edition of Equinox (#10) was never merged, and that approach no longer works due to code/template updates.

marumari commented:
Hmm, I don't think I ever merged this in the textual5 version either.  I'll play around with this and merge it when I get a chance.  :)

dgw commented:
Never merged, but also never closed/refused, so I figured as long as I was updating it to work in Textual 6 for myself I'd share it. :) It's a completely rewritten approach anyway.

dgw commented:
Hmm, this doesn't seem to apply when Textual loads backlog. Is that worth trying to fix?

dgw commented:
When Textual reloaded the style just now (I pulled in recent changes from here and rebased my custom branch), it applied the brackets to the history. I restarted to test whether it worked on history loaded at startup, too, and the brackets were also added there. I will strike my earlier comment about this not working.

dgw commented:
I rarely use Textual any more, and version 6 is an old release at this point. This theme hasn't been updated in nearly two years, either. Closing to reduce clutter on my personal PR dashboard.
